bin/check-bins.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+#
+# Usage: bin/check-bins.sh PACKAGE_INSTALL_PATH
+#
+# Finds any executable or library files that are missing essential
+# dependencies.
+#
+set -euo pipefail
+
+_header_printed=""
+
+error() {
+    echo "$@" >&2
+}
+
+usage() {
+    error "Usage: $0 PACKAGE_INSTALL_PATH"
```

Comment:
Rather than an install path, would it make sense to pass it a package ident?   The way I would I would envision using this is `source results/last_build.env; bin/check-bins.sh $pkg_ident` or `hab pkg install <some_ident_to_test>; bin/check-bins.sh <that same ident>`.